### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by dshchedr** [cyb3rdog/escapepod-docker#1]
Hey, one quick question - should I buy "escape pod" from DDL for using this docker?

**Comment by cyb3rdog** [comment]
Yes, buying the 'escape pod' from DDL will actually provide you with a BASE64 encoded License, bound to your Vector's serail number. Using the escapepod software requires your robot to be licensed, regardless of whether you run it on the rpi, or in docker. 
This docker image is actually just a dockerization of the distribution files and software, and it is recommended to use it after your robot is already onboarded, as the onboarding is not fully supported in this docker image, due to docker's bluetooth capabilities limitations.
